### DISCUSSION TITLE
Import Orders to Craft Commerce, from Craft Commerce

### DISCUSSION THREAD

**Discussion by matthisamoto** [craftcms/commerce#2365] [Ideas]
As far as I can tell there is no way to import orders that are exported via the Orders index page?

The Sprout Import plugin apparently [was not working](https://github.com/craftcms/commerce/issues/375) with Orders previously and [is now discontinued](https://github.com/barrelstrength/craft-sprout-import/issues/74). Feed Me only handles Commerce Products, not Orders.

Am I missing something? The use-case for this is migrating orders from one Craft store to another, as it is replacing the older one.

**Comment by lukeholder** [comment]
There is no order import tool available at the moment.

When we add support I think it would be through Feedme. https://github.com/craftcms/feed-me/issues/472

**Comment by dwhoban** [comment]
@lukeholder is there a way to do it by just importing the DB tables for Orders, Transactions etc but otherwise starting with a clean instance?

**Comment by bymayo** [comment]
Just a heads up. Because this functionality doesn't seem to be arriving any time soon for Craft Commerce / Feed Me - I've created a plugin to handle this in the meantime - https://github.com/bymayo/craft-order-importer
